src/ClimaAtmos/Parameterizations/TurbulenceConvection/src/StateVecs/runtests.jl

```diff
@@ -0,0 +1,75 @@
+using Test
+
+using CLIMA.TurbulenceConvection.Grids
+using CLIMA.TurbulenceConvection.StateVecs
+using CLIMA.TurbulenceConvection.GridOperators
+using CLIMA.TurbulenceConvection.BoundaryConditions
+
+n_subdomains = 3 # number of sub-domains
+n_elems_real = 10 # number of elements
+
+grid = Grid(0.0, 1.0, n_elems_real)
+vars = ( (:ρ_0, 1), (:a, n_subdomains), (:w, n_subdomains) )
+state_vec = StateVec(vars, grid)
+
+@testset "Memory access" begin
+  state_vec[:ρ_0, 1] = 2.0
+  @test state_vec[:ρ_0, 1] == 2.0
+
+  state_vec[:w, 1, 1] = 3.0
+  @test state_vec[:w, 1, 1] == 3.0
+
+  @test_throws BoundsError state_vec[:w, 1, 4] = 3.0
+  @test_throws BoundsError state_vec[:ρ_0, 1, 2] = 3.0
+
+  @test try
```

Comment:
can remove the `@test`: the code will run an error will cause a test failure anyway